manifests/i/IntelCorporation/IntelSystemSupportUtility/2.5.0.15/IntelCorporation.IntelSystemSupportUtility.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: IntelCorporation.IntelSystemSupportUtility

```

Intel Corporation have few softwares under Intel so its better to add it there
```suggestion
PackageIdentifier: Intel.IntelSystemSupportUtility
```